great_expectations/data_context/types/base.py

```diff
@@ -254,6 +254,12 @@ class Meta:
     schema_name = fields.String(required=False, allow_none=True)
     batch_spec_passthrough = fields.Dict(required=False, allow_none=True)
 
+    # Necessary addition for AWS Glue Data Catalog assets
```

Comment:
@lccasagrande Minor: it would be nice to show a link or a brief description as to the reasons that these two fields are important.  Thanks!

Reply:
Added some comments